packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -82,6 +82,39 @@ export type CallCompositeOptions = {
    * @defaultValue true
    */
   callControls?: boolean | CallControlOptions;
+  /* @conditional-compile-remove(call-readiness) */
+  /**
+   * Callback you may provide to supply users with further steps to troubleshoot why they have been
+   * unable to grant your site the required permissions for the call.
+   *
+   * @example
+   * ```ts
+   * onPermissionsTroubleshootingClick: () =>
+   *  window.open('https://contoso.com/permissions-troubleshooting', '_blank');
+   * ```
+   *
+   * @remarks
+   * if this is not supplied, the composite will not show a 'further troubleshooting' link.
+   */
+  onPermissionsTroubleshootingClick?: (permissionsState: {
+    camera: PermissionState;
+    microphone: PermissionState;
+  }) => void;
+  /* @conditional-compile-remove(call-readiness) */
+  /**
+   * Callback you may provide to supplu users with further steps to troubleshoot why they have been
+   * having network issues when connecting to the call.
+   *
+   * @example
+   * ```ts
+   * onNetworkingTroubleShootingClick?: () =>
```

Comment:
Since these methods return `void`, the only UX that Contoso can provide here is via:

* navigating away from the call entirely
* Overlaying something on top of the call UI

The first one seems fairly disruptive, and not something I'd advice Contoso to do.

Should we instead accept a function that returns a `JSX.Element` that we overlay on the composite ourselves?

Reply:
So the reason for this is Contoso is to provide their own 'extra' help page. due to the possibilities of issues we cannot provide a solution every single possible reason that the end user is having issues with their camera, mic or network. these API's are to be used in our error bars to send the end user to a help page that is provided by Contoso. This page might talk about common network issues, or a more detailed guide on how to enable camera/mic permissions on system and browser.

Reply:
It's similar to custom button injection, sure they can inject a button - but there's such limited space for Contoso to present UI based on that button click.
I think we should keep this as void - Contoso can then present a modal dialog or open a new tab page as desired.
We can then continue the conversation on customization points - I strongly feel we should be exposing the side pane a custom content point